examples/waveShared.cpp

```diff
@@ -111,9 +111,19 @@ void innerLoop(const serialLoop &, int Ny, int Nx, Func &&innerLoop){
   }
 #endif
 
```

Comment:
@ajkunen , can we delete this file?

Reply:
@artv3 , yeah, those were your examples... I started converting one of them, but then realized they weren't even in the CMakeLists files anymore.
Do you want to remove them as an ancillary PR?